src/server/auth/server/api_server.go

```diff
@@ -253,6 +257,14 @@ func NewAuthServer(
 			nil,
 			nil,
 		),
+		defaultAcl: col.NewCollection(
```

Comment:
I considered making this a special-case in the existing ACLs collection (using some name that's not a valid repo for example) but decided it's better to be explicit and more verbose.